tests/integration/backward_compatible/proxy/multi_map_test.py

```diff
@@ -150,6 +150,15 @@ def test_put_get(self):
 
         self.assertCountEqual(self.multi_map.get("key"), ["value1", "value2"])
 
+    def test_put_all_get(self):
```

Comment:
You need to guard this method with a client version check for backward compatibility tests. See `map_test#test_remove_interceptor` for such a check. You might need to bump the client version for this to work as well

Reply:
Updated, [87a6da9](https://github.com/hazelcast/hazelcast-python-client/pull/573/commits/87a6da958a9cfc1bc57fff31af7995cf7081020e)

Reply:
the added check is not correct. this test is added for the 5.2 version of the client, so the check should be done against this version, not 5.0